docs/add-ons/tutorial.md

```diff
@@ -117,12 +117,12 @@ RUN apk add --no-cache python3
 WORKDIR /data
 ```
 
-Add "ports" to `config.json`. This will make TCP on port 8000 inside the container available on the host on port 8000.
+Add "ports" to `config.json`. This will make TCP on port 8000 inside the container available on the host on port 8000. Please note that the version must be incremented as well. This way the supervisor will know that a new version of the add-on is available. 
 
 ```json
 {
   "name": "Hello world",
-  "version": "0.2",
+  "version": "2",
```

Comment:
Why this change?

Reply:
The tutorial starts with version "1", I believe that it is more coherent to use version "2", or alternatively to change version of the first example to "0.1" and leave the second one as "0.2". Whichever makes more sense. 
Regards! 
B